Triangulation/include/CGAL/TDS_full_cell_mirror_storage_policy.h

```diff
@@ -42,7 +42,7 @@ struct   TFC_data< Vertex_handle, Full_cell_handle, Maximal_dimension, TDS_full_
 
     void set_mirror_index(const int i, const int index)
     {
-        mirror_vertices_[i] = index;
+        mirror_vertices_[i] = static_cast<std::int_least8_t>(index);
```

Comment:
triggers:
```
/mnt/testsuite/include/CGAL/TDS_full_cell_mirror_storage_policy.h:45:29: warning: writing 1 byte into a region of size 0 [-Wstringop-overflow=]
   45 |         mirror_vertices_[i] = static_cast<std::int_least8_t>(index);
   ```
   [here](https://cgal.geometryfactory.com/CGAL/testsuite/CGAL-6.0-Ic-173/Triangulation/TestReport_lrineau_Ubuntu-GCC_master_CXX20-Release.gz)

Reply:
-Wstringop-overflow and -Warray-bounds are 2 of the worst offenders among the recent gcc warnings that only ever give false positives...